### PR TITLE
Pre-migration formatting fixes

### PR DESCRIPTION
**Pull request by kentdanas**
A couple of minor formatting fixes found when reviewing the `guide-migration-base` deploy preview.

### CODE REVIEW

File: learn/airflow-azure-data-factory-integration.md [hunk 1]
```diff
@@ -8,7 +8,11 @@ tags: ["Integrations", "Azure"]
 
 Azure Data Factory (ADF) is a commonly used service for constructing data pipelines and jobs. With a little preparation, it can be used in combination with Airflow to leverage the best of both tools. In this guide, we'll discuss why you might want to use these two tools together, how Airflow can be used to execute ADF jobs, and a simple example tutorial showing how it all fits together.
 
-> **Note:** All code in this guide can be found on [the Astronomer Registry](https://registry.astronomer.io/dags/azure-data-factory-dag).
+:::note
```
Comment (jwitz) [inline learn/airflow-azure-data-factory-integration.md:11]:
```suggestion
:::info
```

File: learn/airflow-azure-data-explorer.md [hunk 2]
```diff
@@ -10,7 +10,11 @@ tags: ["Integrations", "Azure", "DAGs"]
 
 You can use the ADX [Hook](https://registry.astronomer.io/providers/microsoft-azure/modules/azuredataexplorerhook) and [Operator](https://registry.astronomer.io/providers/microsoft-azure/modules/azuredataexplorerqueryoperator) which are part of the Azure provider package, to integrate ADX queries into your DAGs. In this guide, you'll learn how to make your ADX cluster work with Airflow and walk through an example DAG that runs a query against a database in that cluster.
 
-> **Note:** All code in this guide can be found on [the Astronomer Registry](https://registry.astronomer.io/dags/azure-data-explorer-tutorial).
+:::note
```
Comment (jwitz) [inline learn/airflow-azure-data-explorer.md:13]:
```suggestion
:::info
```

File: learn/airflow-databricks.md [hunk 3]
```diff
@@ -10,7 +10,11 @@ tags: [Integrations, DAGs]
 
 Astronomer has many customers who use Databricks to run jobs as part of complex pipelines. This can easily be accomplished by leveraging the [Databricks provider](https://registry.astronomer.io/providers/databricks), which includes Airflow hooks and operators that are actively maintained by the Databricks and Airflow communities. In this guide, you'll learn about the hooks and operators available for interacting with Databricks clusters and run jobs, and how to use both available operators in an Airflow DAG.
 
-> **Note**: All code in this guide can be found on [the Astronomer Registry](https://registry.astronomer.io/dags/databricks-tutorial).
+:::note
```
Comment (jwitz) [inline learn/airflow-databricks.md:13]:
```suggestion
:::info
```

File: learn/airflow-dbt.md [hunk 4]
```diff
@@ -18,7 +18,11 @@ In this guide, you'll:
     - At the [model](https://docs.getdbt.com/docs/building-a-dbt-project/building-models) level. 
 - Learn how to extend the model-level use case by automating changes to a dbt model.
 
-> The dbt Core sections of this guide summarize some of the key practices and findings from our blog series with [Updater](https://updater.com/) and [Sam Bail](https://sambail.com/) about using dbt in Airflow. For more information, check out [Part 1](https://www.astronomer.io/blog/airflow-dbt-1), [Part 2](https://www.astronomer.io/blog/airflow-dbt-2), and [Part 3](https://www.astronomer.io/blog/airflow-dbt-3) of the series.
+:::note
```
Comment (jwitz) [inline learn/airflow-dbt.md:21]:
```suggestion
:::info
```

File: learn/airflow-dbt.md [hunk 5]
```diff
@@ -99,7 +103,11 @@ dag = check_before_running_dbt_cloud_job()
 
 In the `DbtCloudRunJobOperator` you must provide the dbt connection ID as well as the `job_id` of the job you are triggering.
 
-> **Note:** The full code for this example, along with other DAGs that implement the dbt Cloud Provider, can be found on the [Astronomer Registry](https://registry.astronomer.io/dags?providers=dbt+Cloud&page=1).
+:::note
```
Comment (jwitz) [inline learn/airflow-dbt.md:106]:
```suggestion
:::info
```

File: learn/airflow-dbt.md [hunk 6]
```diff
@@ -233,7 +241,11 @@ for node in data["nodes"].keys():
                 dbt_tasks[upstream_node] >> dbt_tasks[node]
 ```
 
-> **Note:** The code for this example can be found on [the Astronomer Registry](https://registry.astronomer.io/dags/dbt-advanced).
+:::note
```
Comment (jwitz) [inline learn/airflow-dbt.md:244]:
```suggestion
:::info
```

File: learn/airflow-dbt.md [hunk 7]
```diff
@@ -398,7 +410,11 @@ To add this functionality, you can take a group of models defined by some select
            dbt_tasks[edge[0]] >> dbt_tasks[edge[1]]
   ```
 
-  > **Note:** The functions in the DAG file above have been split out for simplicity, but the logic can be found in the [dbt_advanced.py DAG](https://github.com/astronomer/airflow-dbt-demo/blob/master/dags/dbt_advanced.py).
+:::note
```
Comment (jwitz) [inline learn/airflow-dbt.md:413]:
```suggestion
:::info
```

File: learn/airflow-great-expectations.md [hunk 8]
```diff
@@ -48,7 +52,11 @@ If you want to use your own Great Expectations project along with this guide, en
 
 If you set up a project manually, you will see a `great_expectations` directory which contains several sub-directories, as well as the `great_expectations.yml` configuration file. If you cloned the demo repository, the `great_expectations` directory can be found under `include/`.
 
-> Note: The `GreatExpectationsOperator` requires Airflow 2.1 or later, and you will need to change the value of `enable_xcom_pickling` to `true` in your airflow.cfg. If you are using an Astronomer project structure, add `ENV AIRFLOW__CORE__ENABLE_XCOM_PICKLING=True` to your Dockerfile. If you are working from the demo repository, this step has already been completed for you.
+:::note
```
Comment (jwitz) [inline learn/airflow-great-expectations.md:55]:
```suggestion
:::info
```

File: learn/airflow-great-expectations.md [hunk 9]
```diff
@@ -48,7 +52,11 @@ If you want to use your own Great Expectations project along with this guide, en
...
+
+The `GreatExpectationsOperator` requires Airflow 2.1 or later, and you will need to change the value of `enable_xcom_pickling` to `true` in your airflow.cfg. If you are using an Astronomer project structure, add `ENV AIRFLOW__CORE__ENABLE_XCOM_PICKLING=True` to your Dockerfile. If you are working from the demo repository, this step has already been completed for you.
```
Comment (jwitz) [inline learn/airflow-great-expectations.md:57]:
```suggestion
The `GreatExpectationsOperator` requires Airflow 2.1 or later, and you will need to change the value of `enable_xcom_pickling` to `true` in your `airflow.cfg` file. If you are using an Astronomer project structure, add `ENV AIRFLOW__CORE__ENABLE_XCOM_PICKLING=True` to your Dockerfile. If you are working from the demo repository, this step has already been completed for you.
```

File: learn/airflow-openlineage.md [hunk 10]
```diff
@@ -12,7 +12,11 @@ It follows that data lineage has a natural integration with Apache Airflow. Airf
 
 In this tutorial, you’ll define data lineage, review the OpenLineage standard and core lineage concepts, describe why you would want data lineage with Airflow, and implement an example local integration of Airflow and OpenLineage for tracking data movement in Postgres.
 
-> Note: This guide focuses on using OpenLineage with Airflow 2. Some topics may differ if you are using it with earlier versions of Airflow.
+:::note
```
Comment (jwitz) [inline learn/airflow-openlineage.md:15]:
```suggestion
:::info
```

File: learn/airflow-passing-data-between-tasks.md [hunk 11]
```diff
@@ -6,12 +6,16 @@ id: airflow-passing-data-between-tasks
 tags: [DAGs, XCom, Tasks, Dependencies]
 ---
 
-> Note: All code in this guide can be found in [the Github repo](https://github.com/astronomer/airflow-guide-passing-data-between-tasks).
-
 Sharing data between tasks is a very common use case in Airflow. If you've been writing DAGs, you probably know that breaking them up into smaller tasks is a best practice for debugging and recovering quickly from failures. What do you do when one of your downstream tasks requires metadata about an upstream task, or processes the results of the task immediately before it?
 
 There are a few methods you can use to implement data sharing between your Airflow tasks. In this tutorial, you'll walk through the two most commonly used methods, learn when to use them, and use some example DAGs to understand how they can be implemented.
 
+:::note
```
Comment (jwitz) [inline learn/airflow-passing-data-between-tasks.md:13]:
```suggestion
:::info
```

**Review by jwitz — APPROVED** [review]
Just minor changes to the formatting, otherwise LGTM!

File: learn/airflow-azure-data-explorer.md (hunk 2 above)
Comment (kwcanuck) [reply learn/airflow-azure-data-explorer.md:13]: Is it necessary to enclose this information in an info box? Is it that significant? Overuse of info boxes leads to visual clutter and makes documentation hard to scan.

**Review by kwcanuck — APPROVED** [review]

File: learn/airflow-azure-data-explorer.md (hunk 2 above)
Comment (jwitz) [reply learn/airflow-azure-data-explorer.md:13]: I agree, but let's hold off on making any decisions for these boxes until after the migration completes.

**Pull request merged**